devel/ddev/Portfile

```diff
@@ -0,0 +1,335 @@
+                    gopkg.in/yaml.v3 v3.0.1 \
+                        h1:fxVm/GzAzEWqLHuvctI91KS9hhNmmWOoWu0XTYJS7CA= \
+                    gotest.tools/v3 v3.4.0 \
+                        h1:ZazjZUfuVeZGLAmlKKuyv3IKP5orXcwtOwDQH6YVr6o= \
+                    muzzammil.xyz/jsonc v1.0.0 \
+                        h1:B6kaT3wHueZ87mPz3q1nFuM1BlL32IG0wcq0/uOsQ18=
+
+# Dependencies
+depends_build       port:go \
+                    port:gmake
+
+depends_run         port:mkcert
+
+# Build configuration
+build.env-append    CGO_ENABLED=0 \
+                    GOFLAGS="-mod=vendor -buildvcs=false"
+
+build.cmd           make
+
+# Set build target based on architecture
+if {${build_arch} eq "x86_64"} {
+    set go_arch "amd64"
+} elseif {${build_arch} eq "arm64"} {
+    set go_arch "arm64"
+}
+
+build.target    darwin_${go_arch}
+
+# Initialize a git repository because ddev's build process uses git information
+# for version detection and embedding build information in the binary
+post-extract {
+    system -W ${worksrcpath} "git init"
+    system -W ${worksrcpath} "git config user.email 'macports@localhost'"
+    system -W ${worksrcpath} "git config user.name 'MacPorts Build'"
+    system -W ${worksrcpath} "git add ."
+    system -W ${worksrcpath} "git commit -m 'Initial commit'"
+    system -W ${worksrcpath} "git tag v${version}"
+}
+
+build {
+    system -W ${worksrcpath} "git config --global --add safe.directory ${worksrcpath}"
+    system -W ${worksrcpath} "${build.cmd} ${build.target}"
+}
+
+destroot {
+    # Install binary
+    xinstall -m 755 ${worksrcpath}/.gotmp/bin/darwin_${go_arch}/ddev ${destroot}${prefix}/bin/
+    
+    # Generate shell completions
+    system -W ${worksrcpath} "git config --global --add safe.directory ${worksrcpath}"
+    system -W ${worksrcpath} "GOFLAGS=-buildvcs=false ${build.cmd} completions"
+    
+    # Extract completions
+    system -W ${worksrcpath} "mkdir -p .gotmp/bin/completions_extracted"
+    system -W ${worksrcpath} "tar xzf .gotmp/bin/completions.tar.gz -C .gotmp/bin/completions_extracted"
+    
+    # Install shell completions
+    xinstall -d ${destroot}${prefix}/share/bash-completion/completions
+    xinstall -m 644 ${worksrcpath}/.gotmp/bin/completions_extracted/ddev_bash_completion.sh \
+        ${destroot}${prefix}/share/bash-completion/completions/ddev
+        
+    xinstall -d ${destroot}${prefix}/share/zsh/site-functions
+    xinstall -m 644 ${worksrcpath}/.gotmp/bin/completions_extracted/ddev_zsh_completion.sh \
+        ${destroot}${prefix}/share/zsh/site-functions/_ddev
+        
+    xinstall -d ${destroot}${prefix}/share/fish/vendor_completions.d
+    xinstall -m 644 ${worksrcpath}/.gotmp/bin/completions_extracted/ddev_fish_completion.sh \
+        ${destroot}${prefix}/share/fish/vendor_completions.d/ddev.fish
+}
+
+test.run            yes
+test.cmd            make
+test.target         test
+
+livecheck.type      regex
```

Comment:
You should be able to replace all these `livecheck` lines with just `github.livecheck.regex`